include/groonga/table.h

```diff
@@ -505,6 +505,26 @@ struct _grn_table_sort_key {
   int offset;
 };
 
+/**
+ * \brief Sort the records in table and store the results in `result`
+ *
+ * \param ctx The context object
+ * \param table Target table
+ * \param offset Starting offset of the sorted record.
+ *               The records are stored in order from the offset (zero-based).
+ * \param limit Maximum number of records to be stored in `result`
+ * \param result Table to store results
+ * \param keys Array of sort keys
+ *             * keys.key: You can specify either column, accessor, or proc of
+ *               table
+ *             * keys.flags: You can specify the next
+ *               * GRN_TABLE_SORT_ASC: Sort in ascending order
```

Comment:
How about using `\ref` here like the following PR?
- https://github.com/groonga/groonga/pull/1868
```suggestion
 *               * \ref GRN_TABLE_SORT_ASC
```